test/AlarmClockTest.cpp

```diff
@@ -9,17 +9,15 @@
 #include "StopWatch.h"
 #include <chrono>
 
-// The timing leeway was determined
-// experimentally. It takes around 130-160 
-// microseconds on average to get through
-// one while loop of 500ms sleep. 
-int kTimingLeeway = 250;
+unsigned int AlarmClockTest::mFakeSleepUs = 0;
```

Comment:
@john-gress I think this is safe (if not technically "thread-safe"). I only change this variable from the AlarmClock background thread, and I only read this variable after I have asserted that ```alarmClock.Expired()```.

@KjellKod could you weigh in on this?